changelog.md

```diff
@@ -1,3 +1,144 @@
+* lightrider: serialize errors in artifacts ([#9410](https://github.com/googlechrome/lighthouse/pull/9410))
+
+## Deps
+
+* update `axe-core` to 3.3.0 ([#9343](https://github.com/googlechrome/lighthouse/pull/9343))
+* update `typescript` to 3.5.3 ([#9357](https://github.com/googlechrome/lighthouse/pull/9357))
+* update `chrome-launcher` to 0.11.1 ([#9339](https://github.com/googlechrome/lighthouse/pull/9339))
+* `intl`: move from devDep to dep ([#9309](https://github.com/googlechrome/lighthouse/pull/9309))
+* update `debug` to 2.6.9 ([#9398](https://github.com/googlechrome/lighthouse/pull/9398))
+* update to `open` from `opn` ([#9267](https://github.com/googlechrome/lighthouse/pull/9267))
+* update outdated transitive deps ([#9347](https://github.com/googlechrome/lighthouse/pull/9347))
+* update `brfs` to 2.0.2 ([#9312](https://github.com/googlechrome/lighthouse/pull/9312))
+* remove `commitizen` as local dep ([#9246](https://github.com/googlechrome/lighthouse/pull/9246))
+* update `extend` to 3.0.2 ([#9184](https://github.com/googlechrome/lighthouse/pull/9184))
+* bump transitive dep `js-yaml` ([#9131](https://github.com/googlechrome/lighthouse/pull/9131))
+* update `bundlesize`, add more budgets ([#9089](https://github.com/googlechrome/lighthouse/pull/9089))
+
+## Docs
+
+* add i18n overview and authoring documentation ([#9361](https://github.com/googlechrome/lighthouse/pull/9361))
+* link to good first issues ([#9440](https://github.com/googlechrome/lighthouse/pull/9440))
+* add instructions for testing with self-signed certificate ([#9112](https://github.com/googlechrome/lighthouse/pull/9112))
+* update `understanding-results.md` ([#9226](https://github.com/googlechrome/lighthouse/pull/9226))
+* change score type from 'string' to 'number' ([#9225](https://github.com/googlechrome/lighthouse/pull/9225))
+* readme: add `Exthouse` to related projects ([#9243](https://github.com/googlechrome/lighthouse/pull/9243))
+* readme: add `lighthouse-jest-example` to related projects ([#9172](https://github.com/googlechrome/lighthouse/pull/9172))
+* readme: add `lighthouse-persist` to related projects([#9161](https://github.com/googlechrome/lighthouse/pull/9161))
+* readme: add `gimbal` integration to related projects ([#9083](https://github.com/googlechrome/lighthouse/pull/9083))
+* readme: update `lighthouse-mocha-example` related project ([#9158](https://github.com/googlechrome/lighthouse/pull/9158))
+* readme: alphabetize related projects ([#9110](https://github.com/googlechrome/lighthouse/pull/9110))
+* fix various typos ([#9411](https://github.com/googlechrome/lighthouse/pull/9411))
+
+## I18n
+
+* introduce script to swap in new locale to LHR ([#8755](https://github.com/googlechrome/lighthouse/pull/8755))
+* enable local debug locale, `en-XL` ([#9192](https://github.com/googlechrome/lighthouse/pull/9192))
+* localize audits in `best-practices` ([#9092](https://github.com/googlechrome/lighthouse/pull/9092))
```

Comment:
So they are prepped for translation, but not translated.  Could add a disclaimer: 

`Note: best-practices audits are not yet translated to all languages in this release. Expected to be translated fully next release.`

Reply:
done (but shorter :)